klangbecken.py

```diff
@@ -1041,9 +1043,16 @@ def playlog_cmd(data_dir, filename, off_air=False, dev_mode=False):
         json.dump(data, f, **json_opts)
         del data
 
+    # Update file metadata
+    FileType = SUPPORTED_FILE_TYPES[filename.split(".")[-1]]
+    mutagenfile = FileType(filename)
+    mutagenfile["last_play"] = str(now.timestamp())
+
```

Comment:
Save the file after modifying (`mutagenfile.save()`).

Reply:
Thanks, fixed it.